docs.html

```diff
@@ -66,8 +66,14 @@
 <h2>
 <a name="publishing" class="anchor" href="#publishing"><span class="octicon octicon-link"></span></a>Publishing a module</h2>
 
-<p>Push a signed tag to the module then ping an admin (@daenney, @igalic, @nibalizer) and they will publish the module.</p>
-<p>TODO: document module publish process for admins</p>
+<p>Forge publishing is handled by travis and puppet-blacksmith.</p>
+<p>To enable a module to be pushed, cargo-cult the 'deploy' block from puppetcommunity-extlib travis.yml.</p>
```

Comment:
I would replace the word cargo-cult, I had to look it up on wikipedia, it's a little derogatory, perhaps just use 'copy' instead.